data/transaction.go

```diff
@@ -27,61 +27,9 @@ type Transaction struct {
-	ReceiverUsername                  []byte                                `json:"receiverUsername,omitempty"`
-	GasPrice                          uint64                                `json:"gasPrice,omitempty"`
-	GasLimit                          uint64                                `json:"gasLimit,omitempty"`
-	Data                              []byte                                `json:"data,omitempty"`
-	CodeMetadata                      []byte                                `json:"codeMetadata,omitempty"`
-	Code                              string                                `json:"code,omitempty"`
-	PreviousTransactionHash           string                                `json:"previousTransactionHash,omitempty"`
-	OriginalTransactionHash           string                                `json:"originalTransactionHash,omitempty"`
-	ReturnMessage                     string                                `json:"returnMessage,omitempty"`
-	OriginalSender                    string                                `json:"originalSender,omitempty"`
-	Signature                         string                                `json:"signature,omitempty"`
-	SourceShard                       uint32                                `json:"sourceShard"`
-	DestinationShard                  uint32                                `json:"destinationShard"`
-	BlockNonce                        uint64                                `json:"blockNonce,omitempty"`
-	BlockHash                         string                                `json:"blockHash,omitempty"`
-	NotarizedAtSourceInMetaNonce      uint64                                `json:"notarizedAtSourceInMetaNonce,omitempty"`
-	NotarizedAtSourceInMetaHash       string                                `json:"NotarizedAtSourceInMetaHash,omitempty"`
-	NotarizedAtDestinationInMetaNonce uint64                                `json:"notarizedAtDestinationInMetaNonce,omitempty"`
-	NotarizedAtDestinationInMetaHash  string                                `json:"notarizedAtDestinationInMetaHash,omitempty"`
-	MiniBlockType                     string                                `json:"miniblockType,omitempty"`
-	MiniBlockHash                     string                                `json:"miniblockHash,omitempty"`
-	Timestamp                         int64                                 `json:"timestamp,omitempty"`
-	Status                            transaction.TxStatus                  `json:"status,omitempty"`
-	HyperblockNonce                   uint64                                `json:"hyperblockNonce,omitempty"`
-	HyperblockHash                    string                                `json:"hyperblockHash,omitempty"`
-	Receipt                           *transaction.ApiReceipt               `json:"receipt,omitempty"`
-	ScResults                         []*transaction.ApiSmartContractResult `json:"smartContractResults,omitempty"`
```

Comment:
So, no JSON field had changed, right?

Reply:
Yes are the same 
https://github.com/ElrondNetwork/elrond-go-core/blob/ebce3311fa4cb0661b8347e88e74b4bcdcd93afe/data/transaction/apiTransactionResult.go#L50